src/script/Script.cpp

```diff
@@ -1118,6 +1123,11 @@ ValueType getSystemVar(const script::Context & context, std::string_view name,
 				*fcontent = player.Full_maxlife;
 				return TYPE_FLOAT;
 			}
+
+			if(boost::starts_with(name, "^player_maxmana")) {
+				*fcontent = player.manaPool.max;
```

Comment:
I think this should probably be player.Full_maxmana as player.manaPool.max does not include increased mana from mental attribute buffs received from equipment/spells/cheats. We should really swap that around at some point so that player.manaPool.max and player.lifePool.max really are the max.

Reply:
Sweet, I'll update the code in a bit! `player.Full_maxmana` could be part of `player.manaPool` as something like `player.manaPool.maxWithBoosts`.

Reply:
Do all other entities have the Full_maxmana property too? `^maxmana` utilizes the same `manaPool.max` property, which could be updated similarly.